workflows/iuf/operations/extract-release-distributions.yaml

```
@@ -49,8 +49,15 @@ spec:
           productName="{{=jsonpath(inputs.parameters.global_params, '$.product_manifest.current_product.name')}}"
           echo "Extract from: $tarfile for product: $productName"
           dir=$(echo "$tarfile" | sed 's|\(.*\)/.*|\1|')
+          output=${tarfile%.tar.*}
+          echo ${output} > /tmp/parent_directory
```

Not the biggest deal, but why not call the variable the same as the name of the file it's being stored to?

```suggestion
          parent_directory=${tarfile%.tar.*}
          echo ${parent_directory} > /tmp/parent_directory
```